modules/lb-vshn-hiera/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "router_ip_addresses" {
+  type        = list(string)
+  description = "Private IPV4 addresses of nodes running ingress routers"
```

Comment:
Note: this doesn't have to be IPs, we just use the IPs at the moment since we don't create DNS records for the nodes.